scripts/update-browser-releases/bun.ts

```diff
@@ -0,0 +1,347 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import fs from 'node:fs/promises';
+import path from 'node:path';
+
+import chalk from 'chalk-template';
+import { compareVersions } from 'compare-versions';
+
+import { spawn } from '../../utils/index.js';
+import stringify from '../lib/stringify-and-order-properties.js';
+
+import {
+  createOrUpdateBrowserEntry,
+  gfmNoteblock,
+  updateBrowserEntry,
+} from './utils.js';
+
+interface GitHubRelease {
+  tag_name?: string;
+  name?: string;
+  draft?: boolean;
+  prerelease?: boolean;
+  published_at?: string; // ISO
+  html_url?: string;
+}
+
+const GITHUB_API =
+  'https://api.github.com/repos/oven-sh/bun/releases?per_page=100&page=';
+
+const SEMVER_RE = /(\d+\.\d+\.\d+)/;
+
+/**
+ * Converts an ISO date string to a YYYY-MM-DD format.
+ * @param iso - The ISO date string to convert.
+ * @returns The date in YYYY-MM-DD format, or undefined if input is falsy.
+ */
+const toDate = (iso?: string): string | undefined =>
+  iso ? new Date(iso).toISOString().split('T')[0] : undefined;
+
+/**
+ * Extracts the version number from a GitHub release.
+ * @param r - The GitHub release object.
+ * @returns The version number, or undefined if not found.
+ */
+const extractVersion = (r: GitHubRelease): string | undefined => {
+  const hay = `${r.tag_name ?? ''} ${r.name ?? ''}`;
+  const m = hay.match(SEMVER_RE);
+  return m ? m[1] : undefined;
+};
+
+/**
+ * Fetches all Bun GitHub releases across pages.
+ * @returns A list of GitHub release objects.
+ */
+const fetchAllReleases = async (): Promise<GitHubRelease[]> => {
+  const all: GitHubRelease[] = [];
+  for (let page = 1; page <= 10; page++) {
+    const res = await fetch(GITHUB_API + page, {
+      headers: {
+        'User-Agent':
+          'MDN-Browser-Release-Update-Bot/1.0 (+https://developer.mozilla.org/)',
+        Accept: 'application/vnd.github+json',
+      },
+    });
+    if (!res.ok) {
+      throw new Error(`GitHub releases fetch failed: HTTP ${res.status}`);
+    }
+    const chunk = (await res.json()) as GitHubRelease[];
+    if (!Array.isArray(chunk) || chunk.length === 0) {
+      break;
+    }
+    all.push(...chunk);
+  }
+  return all;
+};
+
+/**
+ * Resolves the Bun release asset suffix for the current platform/arch.
+ * @returns The asset suffix like "darwin-aarch64" or "linux-x64".
+ */
+const resolveAssetSuffix = () => {
+  const plat = process.platform === 'darwin' ? 'darwin' : process.platform;
```

Comment:
This looks equivalent to the following:
```suggestion
  const plat = process.platform;
```